src/reducers/recipeReducer.js

```diff
@@ -0,0 +1,22 @@
+import { GET_RECIPE } from '../actions/action-types';
+
+const initialState = {
+  recipes: [
```

Comment:
Would this be the best structure for this application's store? Re: [the Redux docs](https://redux.js.org/recipes/structuring-reducers/normalizing-state-shape)